gold/testsuite/nanomips_long_balc_bc_offset.s

```diff
@@ -0,0 +1,19 @@
+	.linkrelax
+	.section	.text,"ax",@progbits
+	.align	4
```

Comment:
Note that these function alignments limit your ability to test right at the limit of the permissible ranges.